pasystem/pasystem-impl/impl/src/resources/db/migration/mysql/V2__create_pasystem.sql

```diff
@@ -0,0 +1,61 @@
+-- Popup screens
+
+CREATE TABLE `pasystem_popup_screens` (
+  `uuid` VARCHAR(255) PRIMARY KEY,
+  `descriptor` VARCHAR(255),
+  `start_time` BIGINT,
+  `end_time` BIGINT,
+  `open_campaign` int(1) DEFAULT NULL,
+  INDEX `start_time` (`start_time`),
+  INDEX `descriptor` (`descriptor`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `pasystem_popup_content` (
+  `uuid` varchar(255) PRIMARY KEY,
+  `template_content` MEDIUMTEXT,
+  FOREIGN KEY (uuid) REFERENCES pasystem_popup_screens(uuid)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `pasystem_popup_assign` (
+  `uuid` varchar(255),
+  `user_eid` varchar(255) DEFAULT NULL,
+   FOREIGN KEY (uuid) REFERENCES pasystem_popup_screens(uuid),
+   INDEX `user_eid` (`user_eid`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `pasystem_popup_dismissed` (
+  `uuid` varchar(255),
+  `user_eid` varchar(255) DEFAULT NULL,
+  `state` varchar(50) DEFAULT NULL,
+  `dismiss_time` BIGINT,
+   UNIQUE KEY `unique_popup_dismissed` (`user_eid`,`state`, `uuid`),
+   FOREIGN KEY (uuid) REFERENCES pasystem_popup_screens(uuid),
+   INDEX `user_eid` (`user_eid`),
+   INDEX `state` (`state`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- Banners
+CREATE TABLE pasystem_banner_alert
```

Comment:
Table case needs to be consistent. If you create as lowercase, code should refer to table in lowercase.

Linux, and thus MySQL on Linux, is case-sensitive by default.